modbus/tools/README.md

```diff
@@ -0,0 +1,129 @@
+# Generate a modbus read class
```

Comment:
Please move this to a README.DEV.md or something and add a readme for users as this will be shown on the website in the users section.